lib/samuel/dispatchers.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Samuel.Dispatchers do
+  @moduledoc """
+  Dispatchers listen to the reports from Checks and act on them.
+  """
+
+  def process_actions(actions) do
+    dispatchers = [
```

Comment:
Since this is a constant we can use a module attribute. Feels like it might be optimised by the compiler. I might be talking shit though.